src/commands/new.js

```diff
@@ -54,6 +55,13 @@ module.exports = function (projectName) {
 
   // Project name set, install in current working directory
   copyTo(path.join(process.cwd(), projectName));
+  _printInstructions(projectName);
   return true;
 };
 
+function _printInstructions(projectName) {
+  logger.info(`${highlight("New GlueStick project created")} at ${filename(process.cwd())}`);
+  logger.info("To run your app and start developing");
+  logger.info(`    cd ${projectName}`);
```

Comment:
should we add a line after this that says `     gluestick start`, before they can get anything in their browser?